src/uebung_3_2/uebung_3_2.java

```diff
@@ -0,0 +1,59 @@
+package uebung_3_2;
+
+public class uebung_3_2 {
+
+	public static void main(String[] args) {
+		double bestellwert = 5000;
+		double bisherigesKundenBestellvolumen = 200000;
+		double rabatt = 0;
+		boolean istPremiumKunde = true;
+
+		// Bestellwertkontrolle
+		if (bestellwert >= 100 && bestellwert < 500) {

```

Comment:
Wenn Sie sich "von oben nach unten" arbeiten würden, bei den Rabatten, wären die Bedingungen in den ```if```s etwas einfacher, weil Sie nicht jeweils die Grenze darüber checken müssten. Ist aber trotzdem ok so.